Sentry.podspec

```diff
@@ -30,7 +30,7 @@ Pod::Spec.new do |s|
 
   s.subspec 'Core' do |sp|
       sp.source_files = "Sources/Sentry/**/*.{h,hpp,m,mm,c,cpp}",
-        "Sources/SentryCrash/**/*.{h,hpp,m,mm,c,cpp}", "Sources/SentrySwift/**/*.{swift}"
+        "Sources/SentryCrash/**/*.{h,hpp,m,mm,c,cpp}", "Sources/Swift/**/*.{swift}"
```

Comment:
`h`: Why do we have such a change in this PR?

Reply:
Good question.

I think is a merge problem.